devtools/server/actors/replay/auth.js

```diff
@@ -233,14 +233,20 @@ function initializeRecordingWebChannel() {
   );
   const localUrl = "http://localhost:8080/";
 
+  const replayio = /^https:\/\/.+.replay.io\/?$/;
+  const previewBranches = /^https:\/\/.+-recordreplay.vercel.app\/?$/;
   // custom subdomains
-  registerWebChannel(/^https:\/\/.+.replay.io$/);
+  registerWebChannel(replayio);
   // preview branches
-  registerWebChannel(/^https:\/\/.+-recordreplay.vercel.app$/);
-  registerWebChannel(pageUrl);
+  registerWebChannel(previewBranches);
+  if (!replayio.test(pageUrl) && !previewBranches.test(pageUrl)) {
+    console.log(pageUrl, replayio, previewBranches);
+    registerWebChannel(pageUrl);
+  }
   registerWebChannel(localUrl);
 
   function registerWebChannel(url) {
+    console.log("Registering WebChannel for", url);
```

Comment:
I'm guessing these console logs don't end up polluting the users pane because they run in a separate browser process?

Reply:
Correct. They only log to the browser console and not the web page's console.